### ISSUE TITLE
First run after installation gives an error

### ISSUE THREAD

**Issue by culpinnis** [tschmutzer/kmasker#60]
For me Kmasker just prints: Can not open '' (external path) if I try to run it on a new machine with Kmasker --help

**Comment by culpinnis** [comment]
Fixed in #62

**Comment by culpinnis** [comment]
The first run is still some kind of weird, especially for new/ inexperienced users: 

```sh
./Kmasker --help

 PLEASE NOTE:
 You are writing all large data structures to your home directory [default].
 It is recommended to modify the path for 'PATH_kindex_private'.

 Use the following command: 'Kmasker --set_private_path enter/your/path'

Can not open '' (private path)
```
